### ISSUE TITLE
Simple version system problem

### ISSUE THREAD

**Issue by PatriceTorchet** [abstractspoon/ToDoList_Issues#44]
Win XP

This morning, the pc on which a coleague was working on tasklist have lost the network when TDL wanted to save the list.
After gaining back the network, TDL was unable to save the list because the file was reported to be locked by this very PC.

**Comment by abstractspoon** [comment]
Did you try to save the tasklist while the network was down?
If so, what was the outcome? Message box?

**Comment by PatriceTorchet** [comment]
Problem append in mùorning, I was called only in afternoon :(

**Comment by PatriceTorchet** [comment]
In the morning:
- The PC checked the list.
- The network went down.
- the PC tried to Check-in the List when it timed out.
Later, the user told me about a message earlier.
-I don't remember exactly what I did, I must have tried first to Check out the list, but got a message saying the list was already checked by another PC, but the name of the PC was himself.
- I have also certainly tried later to save the list or check in, and I got the same message.

TDL was not seing that the one owning the list was himself.
